From 830d16589c2192eb5e0e278f8d68d24ac120ed63 Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Fri, 7 Dec 2012 17:28:30 +0100
Subject: [PATCH 159/204] shutdown: downgrade a warning

All messages of the kind "not all done, %d left" are log_info, except
the one for DM devices. Make it info too.
(cherry picked from commit 2569a5ce16638d99f1ebaaa7774d183496d8b8e8)
---
 src/core/shutdown.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/shutdown.c b/src/core/shutdown.c
index b59aef1..7ec082b 100644
--- a/src/core/shutdown.c
+++ b/src/core/shutdown.c
@@ -233,7 +233,7 @@ int main(int argc, char *argv[]) {
                         if (r == 0)
                                 need_dm_detach = false;
                         else if (r > 0)
-                                log_warning("Not all DM devices detached, %d left.", r);
+                                log_info("Not all DM devices detached, %d left.", r);
                         else
                                 log_error("Failed to detach DM devices: %s", strerror(-r));
                 }
-- 
1.8.1.5

